lib/manageiq-providers-scvmm.rb

```diff
@@ -1,2 +1,10 @@
 require "manageiq/providers/scvmm/engine"
 require "manageiq/providers/scvmm/version"
+
+module ManageIQ
+  module Providers
+    module Microsoft
+      autoload :Discovery, 'manageiq/providers/microsoft/discovery'
```

Comment:
I don't think we need this since `app/models` should already be on the autoload path.

Reply:
Right, thanks